projects/storefrontapp/server.ts

```diff
@@ -75,6 +76,7 @@ function run() {
   // Start up the Node server
   const server = app();
   server.listen(port, () => {
+    /* eslint-disable-next-line no-console */
```

Comment:
Let's add explanation, why we can use console.log in this case. What about?:
```suggestion
    /* eslint-disable-next-line no-console 
        --
        It's just an example application file. This message is not crucial
        to be logged using any special logger. Moreover, we don't have
        any special logger available in this context. */
```